src/DensityInterface.jl

```diff
@@ -1,14 +1,16 @@
 # This file is a part of DensityInterface.jl, licensed under the MIT License (MIT).
 
-__precompile__(true)
-
 """
     DensityInterface
 
 Trait-based interface for mathematical/statistical densities
 """
 module DensityInterface
 
+using InverseFunctions
+using Test
```

Comment:
It's unusual to have Test imported from a file in `src`. Is this intentional?

Reply:
It is not uncommon. This way, we can provide an standard way for users to test API conformity. This also means that in case of future API changes, users will be notified if their density definitions need to be updated. Me and @devmotion discussed the question of depending on `Test` during the design of `InverseFunctions` and `ChangesOfVariables - I was initially hesitant, but it worked out very nicely.

Reply:
It's a common (and IMO very useful) approach to provide a test suite for an interface that checks all required definitions and assumptions. It exists e.g. also in ParameterHandling.jl, DynamicPPL.jl, and KernelFunctions.jl, there in a TestUtils submodule (these packages are much larger than DensityInterface or InverseFunctions and hence a separate module seems a bit cleaner but IMO it is not required here).

Reply:
Interesting, thanks for the background

Reply:
I'll mark this resolved then (please feel free to unresolve if necessary).